pkg/pillar/cmd/zedagent/handleconfig.go

```diff
@@ -507,12 +516,98 @@ func updateCertTimer(configInterval uint32, tickerHandle interface{}) {
 	flextimer.TickNow(tickerHandle)
 }
 
+func decryptAuthContainer(getconfigCtx *getconfigContext, sendRV *zedcloud.SendRetval) error {
+	decryptCtx := &cipher.DecryptCipherContext{
+		Log:                  log,
+		AgentName:            agentName,
+		PubSubControllerCert: getconfigCtx.pubControllerCert,
+		PubSubEdgeNodeCert:   getconfigCtx.zedagentCtx.subEdgeNodeCert,
+	}
+	sm := &zauth.AuthContainer{}
```

Comment:
`sm`? There is neither a `s` nor an `m` in `zauth.AuthContainer`.

Reply:
s-i-g-n-e-d-m-e-s-s-a-g-e

Reply:
```suggestion
	signedMessage := &zauth.AuthContainer{}
```